### ISSUE TITLE
documentation request: reverse change log sort, newest version on top

### ISSUE THREAD

**Issue by kdupke** [j-kaltes/Juggluco#111]
Thank you for working on Juggluco!

I do not exactly know how this repository is linked to/with juggluco.nl, so do not hesitate to redirect me with my request.

https://www.juggluco.nl/Juggluco/changes.html has a change log for Juggluco.

I request to change to order of the entries, so the newest version always is on top.

This change will make it much easier to consume, as it does not need that much scrolling. Many user may be on on of the recent versions so it may fit a single screen on e.g. the phone display.

Thank you for considering this change.

**Comment by j-kaltes** [comment]
Attach a keyboard to your phone and press end.

**Comment by kdupke** [comment]
It looks like you see the benefit. It would mean not to add a keyboard nor to hit a key for most people.

Not sure if you edit this in a manual way or if the release notes are done by a script. Either way it is a a small change that benefits many.

**Comment by tyrex1975** [comment]
Yes, I agree with the OP. Attaching and using a keyboard to navigate to the end is not a user-friendly solution.
